.travis.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       env: PATH=/c/Python38:/c/Python38/Scripts:$PATH
 install: 
     - pip3 install --upgrade pip  # all three OSes agree about 'pip3'
-    - pip3 install black
+    - pip3 install black==19.10b0
```

Comment:
that is the latest black version, which does not destroy our doctests. The new black-version removes some whitespaces because of which the doctests fail. We are keeping track of it in Issue #171